packages/site-server/src/templates/base.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {escapeHTML} from '../catalog/escape-html.js';
+
+export const renderPage = (data: {
+  scripts?: Array<string>;
+  title?: string;
+  content?: string;
+}) => {
+  return `<!doctype html>
```

Comment:
WDYT about a no-op html tag function so that we get syntax highlighting?

Reply:
Seems fine, though I might wait until we integrate lit-ssr and see if we just use that everywhere.